extensions/sql-migration/src/models/strings.ts

```diff
@@ -143,3 +143,9 @@ export const REFRESH = localize('sql.migration.refresh', "Refresh");
 export const SUBMIT = localize('sql.migration.submit', "Submit");
 export const CREATE = localize('sql.migration.create', "Create");
 export const CANCEL = localize('sql.migration.cancel', "Cancel");
+
+
+// Open notebook quick pick string
+export const NOTEBOOK_QUICK_PICK_PLACEHOLDER = localize('sql.migration.quick.pick.placeholder', "Type to search for notebook");
```

Comment:
Search for what? And given there's only two opens "search" seems a little odd. Maybe just something like `Select the operation you'd like to perform` with the options being `Inline Migration` and `SQL Migration Assessment`. Run it by the PMs to see if they have any thoughts.